lib/ansible/playbook/helpers.py

```diff
@@ -207,7 +207,7 @@ def load_list_of_tasks(ds, play, block=None, role=None, task_include=None, use_h
                     try:
                         data = loader.load_from_file(include_file)
                         if data is None:
-                            return []
+                            continue
```

Comment:
i mean `display.warning('file %s was empty and had no tasks to include' % include_file)`